wolfcrypt/src/aes.c

```diff
@@ -356,7 +356,7 @@
 
         HAL_CRYP_DeInit(&hcryp);
     #else
-        #error AES Decrypt not implemented for STM32 StdPeri lib
+        #warning AES Decrypt not implemented for STM32 StdPeri lib
```

Comment:
If this is not implemented what's wrong with a compile time error?  Isn't that better than a compiler warning that will allow code to run without AES encryption/decryption?